test.js

```diff
@@ -331,15 +331,16 @@ function assertRule(t, rule, info) {
  * @param {Rule} info
  * @param {Check} fixture
  * @param {string} basename
- * @param {unknown} settings
+ * @param {Record<string, unknown>} settings
+ * @param {unknown} options
```

Comment:
Maybe use clearer names? E.g., `processorSettings` vs `setting` (or `pluginOption`)

Reply:
- [x] I changed `setting` -> `ruleOptions` and the `{settings, ruleOptions}` container -> `configuration`.